src/components/theme/getRootClassName.ts

```diff
@@ -0,0 +1,10 @@
+import {block} from '../utils/cn';
+import type {CnMods} from '../utils/cn';
+
+import {ROOT_CLASSNAME} from './constants';
+
+const b = block(ROOT_CLASSNAME);
+
+export function getRootClassName(mods: CnMods, className: string) {
```

Comment:
It seems the first parameter is too general, maybe let's make it more strict?

We are not using this method yourself, isn't it?

Reply:
You mean instead of object allow only known keys, like `theme`?

Reply:
Yes